interpreter/src/main/scala/pl/touk/nussknacker/engine/compiledgraph/node.scala

```diff
@@ -43,6 +41,8 @@ object node {
 
   case class EndingCustomNode(id: String, ref: String) extends Node
 
+  case class EndingFragmentDefinition(id: String, fieldsWithExpression: Map[String, TypedExpression], isDisabled: Boolean) extends Node
```

Comment:
`EndingProcessor` and `EndingCustomNode` are special cases of `Processor` and `CustomNode` at the end of scenario :) fragment output definition can be in the middle of valid scenario?

Reply:
Not to confuse those names I've changed `EndingFragmentDefinition` to `FragmentOutput`